tests/scalers/gcp-pubsub.test.ts

```diff
@@ -0,0 +1,235 @@
+import * as crypto from 'crypto'
+import * as fs from 'fs'
+import * as sh from 'shelljs'
+import * as tmp from 'tmp'
+import test from 'ava'
+
+
+const gcpKey = process.env['GCP_SP_KEY']
+const projectId = JSON.parse(gcpKey).project_id
+const testNamespace = 'gcp-pubsub-test'
+const topicId = `projects/${projectId}/topics/keda-test-topic-` + crypto.randomBytes(6).toString('hex')
+const subscriptionName = `keda-test-topic-sub-` + crypto.randomBytes(6).toString('hex')
+const subscriptionId = `projects/${projectId}/subscriptions/${subscriptionName}`
+const deploymentName = 'dummy-consumer'
+const maxReplicaCount = '4'
+const gsPrefix = `kubectl exec --namespace ${testNamespace} deploy/gcp-sdk -- `
+
+test.before(t => {
+    sh.exec(`kubectl create namespace ${testNamespace}`)
```

Comment:
please, use the function `createNamespace(namespace :string)` instead of creating it using kubectl.
The function is inside helpers file (`import { createNamespace } from './helpers';`)

Reply:
We updated [the docs](https://github.com/kedacore/keda/tree/main/tests#adding-tests) yesterday